CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Fixed
+* **dso-toolkit + core:** dso-info in dso-radios mist aria-describedby ([#1399](https://github.com/dso-toolkit/dso-toolkit/issues/1399))
```

Comment:
Het issue heeft het alleen over `dso-info` in `dso-radios`, maar ik zie ook een change in Form Group: Checkboxes voorbij komen.

Ik denk dat de change wel de bedoeling is, maar dan zie ik dat graag terug in de CHANGELOG en titel van het issue